mllib/src/test/scala/org/apache/spark/mllib/tree/RandomForestSuite.scala

```diff
@@ -135,6 +135,23 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(rf1.toDebugString != rf2.toDebugString)
   }
 
+  test("options for feature subset size in RandomForest - SPARK-3724") {
```

Comment:
This test should reside in spark.ML instead of here. Especially since we plan to move DecisionTreeMetadata.scala to ML sometime in the near future.